bin/add-roles.sh

```diff
@@ -7,6 +7,9 @@
 ./bin/utils/ccd-add-role.sh "prd-admin"
 ./bin/utils/ccd-add-role.sh "judge-profile"
 ./bin/utils/ccd-add-role.sh "basic-access"
+./bin/utils/ccd-add-role.sh "admin-access"
+./bin/utils/ccd-add-role.sh "GS_profile"
+./bin/utils/ccd-add-role.sh "legal-adviser"
```

Comment:
Could you please double check whether it is legal-adviser or legal-advisor? Thanks

Reply:
Hi! 
In civil-service and civil-sdk we use legal-adviser.